.eslintrc

```diff
@@ -115,18 +115,21 @@
       "rules": {
         // @TODO: revise these rules
         "@typescript-eslint/consistent-type-assertions": "error",
+        "no-unused-vars": "off",
         "@typescript-eslint/no-unused-vars": [
           "warn",
           {
             "argsIgnorePattern": "^_",
             "varsIgnorePattern": "^_",
-            "caughtErrorsIgnorePattern": "^_"
+            "caughtErrorsIgnorePattern": "^_",
+            "destructuredArrayIgnorePattern": "^_",
+            "ignoreRestSiblings": true
           }
         ],
         "@typescript-eslint/ban-ts-comment": "warn",
         "@typescript-eslint/no-loss-of-precision": "warn",
-        "@typescript-eslint/no-unsafe-declaration-merging": "warn",
-        "react-hooks/exhaustive-deps": "warn",
+        "@typescript-eslint/no-unsafe-declaration-merging": "off",
+        "react-hooks/exhaustive-deps": "off",
```

Comment:
I would vote for leaving this turned off, don't how much of them is there but definitely better would be to suppress them using `eslint-disable` comments. If I recall correctly there isn't much of hook usages.
p.s. Yeah it is bad practice to suppress the warning or error but would always choose that instead of disabling the rule for all future code.

Reply:
My thinking was just having a smaller diff (and I'd like to resolve the `exhaustive-deps` eventually) but you're right better to use disable comments to prevent further instances. Will update